src/examples/track_matching/reco_particles_track_matching.cc

```diff
@@ -14,7 +14,7 @@
 
 using std::string;
 
-void reco_particles_track_matching(char *file_name) {
+void reco_particles_track_matching(TString file_name) {
     auto file = new TFile(file_name);
```

Comment:
```suggestion
void reco_particles_track_matching(std::string &file_name) {
    auto file = new TFile(file_name.c_str());
```

Reply:
Let's not overthink this and let the author, @DraTeots, decide.

Reply:
The most ugliest thing in the code was a string with some file on my system. The second thing, which I don't like about root is that in 2023 we have to use TFile(file_name.**c_str()**);. Since it is a root macro and root is at play, let it be a canonical TString. While I don't really see any difference here and just follow "not overthink" with 50.0001% vs 49.9999% weights =)

Reply:
We should use one string class for everything. If this is a collaboration, authorship doesn't grant ownership over the code.

Reply:
FYI, TFile doesn't have native support for TString, it's just that TString has an implicit conversion operator.